### DISCUSSION TITLE
Pagination ReflectionUnionType::getName

### DISCUSSION THREAD

**Discussion by juangdnar** [filamentphp/filament#7792] [Help]
when i try to change te default pagination number from 10 to another value, as example 50, i get the following error:

ReflectionUnionType::getName

Any tips? thanks in advanced.

**Comment by leandrocfe** [comment]
v3? https://filamentphp.com/docs/3.x/tables/advanced#customizing-the-pagination-options

**Reply by juangdnar** [reply]
yes V3 , im using ->paginated([10, 25, 50, 100, 'all']); in my Resource and i get the Call to undefined method ReflectionUnionType::getName() 

Thanks for the quick reply.

**Reply by leandrocfe** [reply]
all tables?

**Reply by jcgmalave** [reply]
yes, in al teh resources i get the following:

![imagen](https://github.com/filamentphp/filament/assets/13526704/1c63ead1-f952-4c2d-919e-85fefd59b4fe)

**Comment by danharrin** [comment]
#7778
